onyo/lib/commands.py

```diff
@@ -620,12 +620,21 @@ def onyo_mv(inventory: Inventory,
         # MOVE special case
         subject = "mv"
         move_asset_or_dir(inventory, sources[0], destination.parent)
-    elif len(sources) == 1 and sources[0].is_dir() and destination.parent.is_dir():
+    elif len(sources) == 1 and sources[0].is_dir() and destination.parent.is_dir():  # TODO: last condition necessary?
```

Comment:
Thinking about this: the last condition isn't necessary to know what mode we're in (move + rename mode). But having a valid parent is necessary (for all `mv` modes). Perhaps the check is in the wrong location.

Reply:
Yes, location is the point. The underlying `Inventory` method should fail on that (and cheaper than an additional `stat` by `pathlib`). However, need to double-check and doesn't matter for what this PR is about.